src/lib.js

```diff
@@ -53,8 +53,8 @@ module.exports = {
    * @param {string} date
    * @returns {Array}
    */
-  permuteIDs: async (ids, date) => {
-    const salt = await fetchNistBeacon(date)
+  permuteIDs: async (ids, date, tzCode) => {
+    const salt = await fetchNistBeacon(date, tzCode)
```

Comment:
plz add some test cases here with `tzCode`: https://github.com/diracdeltas/accountable-triage/blob/master/test/libTest.js#L49

Reply:
the existing tests for permuteID also need to be fixed cuz they fail without tzCode